src/proto/generation.proto

```diff
@@ -323,8 +324,6 @@ message Request {
     AssetParameters asset = 8;
   }
   optional ConditionerParameters conditioner = 6;
-  optional WeightMethod weight_method = 9;
-  repeated Prompt uc_prompt = 10;
```

Comment:
This change does not introduce any compatibility issues in the proto, however reusing these fields (9,10) may in the future,  probably should mark these fields as reserved https://developers.google.com/protocol-buffers/docs/proto3#reserved if we want to prevent possible breakage.

Reply:
fixed